src/main/scala/io/apibuilder/validation/MultiService.scala

```diff
@@ -77,6 +77,12 @@ trait MultiService extends ResponseHelpers {
     }
   }
 
+  final def getBodyType(operation: ApiBuilderOperation): Option[(String, String)] = {
```

Comment:
let's make this private - unsure of the interface, but if internal it is fine. Generally would prefer a case class to the tuple